base/src/main/java/io/spine/string/MapQuoter.java

```diff
@@ -45,11 +44,11 @@
     @Override
     String quote(String stringToQuote) {
         checkNotNull(stringToQuote);
-        Matcher matcher = compile(QUOTE_PATTERN).matcher(stringToQuote);
-        String unslashed = matcher.find() ?
-                           matcher.replaceAll(BACKSLASH + matcher.group()) :
-                           stringToQuote;
-        String result = QUOTE_CHAR + unslashed + QUOTE_CHAR;
+        var matcher = compile(QUOTE_PATTERN).matcher(stringToQuote);
+        var unslashed = matcher.find() ?
+                        matcher.replaceAll(BACKSLASH + matcher.group()) :
+                        stringToQuote;
```

Comment:
Don't we usually put the `?` and the `:` on the following lines?
I remember that because it does not work in Groovy, we had to use the other format (the one used in this snippet).